### PR TITLE
startTime docs

### PR DESCRIPTION
**Pull request by davidmarkclements**
@Rauno56 does this explain it adequately?

### CODE REVIEW

**Comment by davidmarkclements** [comment]
todo: we should document startTime on `express-pino-logger`, `restify-pino-logger` and `koa-pino-logger`

File: README.md [hunk 1]
```diff
@@ -136,6 +136,26 @@ function handle (req, res) {
 server.listen(3000)
 ```
 
+##### pinoHttp.startTime (Symbol)
+
+The `pinoHttp` function has a property called `startTime` which contains a unique symbol
+which is used attach and reference a start time on the HTTP `res` object. If the function 
+returned from `pinoHttp` is not *the first* function be called in an HTTP servers request
```
Comment (jsumners) [inline README.md]: > ... *the first* function be called ...

^ missing something

Comment (davidmarkclements) [inline README.md]: fixed with force push

**Comment by jsumners** [comment]
Yeah, that's only one of the reasons I don't like force push.

File: README.md [hunk 2]
```diff
@@ -136,6 +136,26 @@ function handle (req, res) {
 server.listen(3000)
 ```
 
+##### pinoHttp.startTime (Symbol)
+
+The `pinoHttp` function has a property called `startTime` which contains a unique symbol
```
Comment (Rauno56) [inline README.md]: Nit: Symbols are unique by definition.

File: README.md [hunk 3]
```diff
@@ -138,7 +138,7 @@ server.listen(3000)
 
 ##### pinoHttp.startTime (Symbol)
 
-The `pinoHttp` function has a property called `startTime` which contains a unique symbol
+The `pinoHttp` function has a property called `startTime` which contains a symbol
 which is used attach and reference a start time on the HTTP `res` object. If the function
```
Comment (jsumners) [inline README.md]: > ... symbol ~which~ that is used ...

Comment (mcollina) [inline README.md]: Fixed

**Pull request merged**